compiler/SymbolicToPure.ml

```
@@ -578,7 +578,7 @@ let translate_type_decl_kind (span : Meta.span) (kind : T.type_decl_kind) :
   | Alias _ ->
       craise __FILE__ __LINE__ span
         "type aliases should have been removed earlier"
-  | T.Opaque -> Opaque
+  | T.Opaque | T.Error _ -> Opaque
```

We should raise an exception in the Error case

---

The exception has already been raised by charon. If we're still here we may as well process as much as possible.

---

Btw this can only happen with `charon --errors-as-warnings`

---

Ok, makes sense